Terminal.Gui/View/Adornment/Border.cs

```diff
@@ -198,6 +198,9 @@ public LineStyle LineStyle
 
     private bool _showTitle = true;
 
+    /// <summary>
+    ///     Gets or sets whether the title should be shown. The default is <see langword="true"/>.
+    /// </summary>
     public bool ShowTitle
```

Comment:
I just had a thought about this that I want to test out at some point.

Since we do not have constructors and are thus using a property-based dependency model, I wonder how these might behave if set to a different value in an initializer, especially if other properties any property setter depends on may have already been set. 🤔

My thinking is that it's a potential for strange NullReferenceExceptions and potential slow resource leakage due to event subscriptions from objects whose references haven't even been pushed onto the evaluation stack yet.

Purely theoretical but making the note both for myself and in case anyone already has specific insight on that behavior in the CLR.